requirements.txt

```diff
@@ -23,4 +23,4 @@ django-fsm==2.7.0
 semantic-version==2.8.5
 django-redis-cache==3.0.0
 sentry-sdk==1.0.0
-kf_lib_data_ingest @ git+https://git@github.com/kids-first/kf-lib-data-ingest.git@1.3.0
+kf_lib_data_ingest @ git+https://git@github.com/kids-first/kf-lib-data-ingest.git@concepts-for-data-tracker
```

Comment:
Maybe we should be tracking a stable release branch instead of a one-off branch?

Reply:
Ohhh yes, this should point to the latest release. And actually this branch should come off of master not `do-validation`